Update/onik_001.txt

```diff
@@ -4186,6 +4186,7 @@ void main()
 	ClearMessage();
 
 //「レナもシカトすんなー！２人がホテル街へ消えたって回覧す＠
+//「レナもシカトすんなー！２人がラブラブだって回覧す < PS3 voice actually says this＠
```

Comment:
Is it possible to at least create a text file with line no./text/whatever that needs changes, so we can deal with it later? This would probably mean to reread the whole thing in Japanese, though, and I doubt anyone wants to do it.